src/js/base/module/Buttons.js

```diff
@@ -388,7 +388,13 @@ export default class Buttons {
           checkClassName: this.options.icons.menuCheck,
           items: this.options.fontSizes,
           title: this.lang.font.size,
-          click: this.context.createInvokeHandlerAndUpdateState('editor.fontSize'),
+          template: function (item) {
+            if (typeof item === 'string') {
+              item = { title: (this.lang.style.hasOwnProperty(item) ? this.lang.style[item] : item), value: item };
+            }
+            return '<span  style="font-size: ' + item.value + 'px;">' + item.title + '</span>';
```

Comment:
Could you keep the original way to show `fontsize` items? (do not apply the `font-size` attribute)
And in this case, we don't have to `this.lang.style` to translate the titles. `this.lang.style` is for paragraph styles, not font size items. So I would like to suggest below;
```javascript
if (typeof item === 'string') {
  // show it as usual
  ...
} else {
  // show with custom tag/template
}
```